### PR TITLE
Inprogress

### PR DESCRIPTION
**Pull request by nickclifford904**

### CODE REVIEW

File: aboutme.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	  <div id="NC"><a href="home.html">Nick Clifford</a> 
+	  </div>
+	  <div class="links">
+		  	<a id="aboutme" href="aboutme.html">About Me</a>
+		  	<a id="portfolio" href="portfolio.html">Portfolio</a>
+		 	<a id="contact" href="contact.html">Contact</a>
+	  </div>
+	  
+  </header>
+	 <div class=aboutmebackground>
```
Comment (dotNATE) [inline aboutme.html]: no quotes for class

File: home.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+		  </div>
+		  <div class="links">
+			  	<a id="aboutme" href="aboutme.html">About Me</a>
+			  	<a id="portfolio" href="portfolio.html">Portfolio</a>
+			 	<a id="contact" href="contact.html">Contact</a>
+		  </div>
+		  
+	  </header>
+	  <main>
+	  <div class=homebackground>
```
Comment (dotNATE) [inline home.html]: no quotes for class

File: homestyle.css [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  text-decoration: none;
+}
+a:focus{
+  color:white;
+  text-decoration: none;
+}
+a:active{
+  color:white;
+  text-decoration: none;
+}
```
Comment (dotNATE) [inline homestyle.css:73]: All a:pseudo-elements could be comma-separated to save repetition

Comment (dotNATE) [reply homestyle.css:73]: If all text is white could be used on universal selector (*)

**Review by dotNATE — COMMENTED** [review]
Very nice look when loaded in browser :)
Only things I spotted are minor syntax slips and some possible refactoring in the css

File: homestyle.css [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+
+.homequote {
+    position: absolute;
```
Comment (mariagus) [inline homestyle.css:89]: Might be good to make this quote move relative to screen size instead, as it stays in one place.

File: aboutme.html [hunk 5]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>About Me</title>
+    <link rel="stylesheet" href="homestyle.css">
```
Comment (mariagus) [inline aboutme.html:6]: If you're using the same style sheet for all your pages, might be clearer to just name it a generic style.css.

**Review by mariagus — COMMENTED** [review]
Hey Nick, it looks good! I like the visuals and existential quote.
My only general note is the random spaces in css formatting.

File: portfolio.html [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+			  	<a id="aboutme" href="aboutme.html">About Me</a>
+			  	<a id="portfolio" href="portfolio.html">Portfolio</a>
+			 	<a id="contact" href="contact.html">Contact</a>
+		  </div>
+		  
+	  </header>
+	  <div class="portmain">
+		  <div class="porttitle">Things I have made...
+		  </div>
+		  <div class=portcontents>
```
Comment (imypullan) [inline portfolio.html:22]: Missing ""

File: portfolio.html [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+		  <div id="NC"><a href="home.html">Nick Clifford</a> 
+		  </div>
+		  <div class="links">
+			  	<a id="aboutme" href="aboutme.html">About Me</a>
+			  	<a id="portfolio" href="portfolio.html">Portfolio</a>
+			 	<a id="contact" href="contact.html">Contact</a>
+		  </div>
+		  
+	  </header>
+	  <div class="portmain">
```
Comment (imypullan) [inline portfolio.html:19]: Lines 19-21 aren't represented in CSS (i.e. no "portmain" or"porttitle") and the text "Things I have made..." don't show up

File: portfolio.html [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+		  <div class="porttitle">Things I have made...
+		  </div>
+		  <div class=portcontents>
+			  <div class="port-container"></div>
+		  </div>
+
+	  </div>
+	<div class="footer">
```
Comment (imypullan) [inline portfolio.html:37]: Do you want your footer to come at the bottom of the page? As currently it comes in as soon as all preceding content is finished.

File: homestyle.css [hunk 9]
```diff
@@ -0,0 +1,176 @@
...
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	padding: 10px;
+	text-decoration: none;
+	letter-spacing: 3px;
+	font-size: 25px;
+}
+
+#NC {
```
Comment (imypullan) [inline homestyle.css:33]: Do you need to have the background-color set as black since the nav bar container is already black?

File: home.html [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+		  <div id="NC"><a href="home.html">Nick Clifford</a> 
+		  </div>
+		  <div class="links">
+			  	<a id="aboutme" href="aboutme.html">About Me</a>
+			  	<a id="portfolio" href="portfolio.html">Portfolio</a>
+			 	<a id="contact" href="contact.html">Contact</a>
+		  </div>	  
+	  </header>
+	  <main>
+	  	<div class=homebackground>
```
Comment (imypullan) [inline home.html]: Missing ""

File: aboutme.html [hunk 11]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
```
Comment (dotNATE) [inline aboutme.html:4]: could also include <meta name="viewport" content="width=device-width, initial-scale=1.0"> but I can't remember exactly what it does...

File: contact.html [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+  	  </header>
+	  
+	  <div class="contactmain">
+		  <div class="contacttitle">How to get in touch
+		  </div>
+		</div>
+		<div class="contactdetails">
+			<div class="contacticons">
+			<div class="mail"><a href="mailto:nickclifford@gmail.com">
+				<img alt="Email" src="mail.png" width="75" height="75">
```
Comment (dotNATE) [inline contact.html:26]: width/height better to set in css

File: contact.html [hunk 13]
```diff
@@ -0,0 +1,61 @@
...
+				</a>
+			</div>
+			<div class="phone"><a href="tel:07970740492">
+				<img alt="Phone" src="phone.png" width="75" height="75">
```
Comment (dotNATE) [inline contact.html:30]: width/height better to set in css

File: contact.html [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+				</a>
+			</div>
+			<div class="twitter"><a href="https://twitter.com/cliffdeford">
+				<img alt="Twitter" src="twitter.png" width="75" height="75">
```
Comment (dotNATE) [inline contact.html:34]: width/height better to set in css

File: contact.html (hunk 13 above)
Comment (dotNATE) [reply contact.html:30]: Also, recruiters would use this as an invitation to call you night and day. Not a code issue but a heads up!

File: homestyle.css [hunk 15]
```diff
@@ -0,0 +1,313 @@
...
+a:active{
+  text-decoration: none;
+}
+
+/* Home */
+
+.homebackground {
+	background: url(home.jpg) no-repeat;
+	width: 90%;
+	height: 70vh;
```
Comment (dotNATE) [inline homestyle.css:80]: I found a lot of problems when testing at different screen heights having used vh. How does this stand up on mobile browsers?

**Comment by dotNATE** [comment]
could use some folders to organise in finder. Looks really good at all screen sizes etc I tried

**Pull request merged**